common/coat_of_arms/coat_of_arms/imperia_coa.txt

```diff
@@ -18,6 +38,16 @@ HNA = {
 	}
 }
 
+HNA_qing = {
```

Comment:
So all these flags don't work anymore with the nice qing flag when qing changes significantly in government. Is there not a way to do this more dynamically, as in fill in the half of the flag with whatever the qing flag is?

Reply:
the only way to dynamically fill flags is via overlord canton type flags, otherwise you'd have to make a variant flag for every flag qing has